app/assets/stylesheets/sign_in.css

```diff
@@ -0,0 +1,116 @@
+h1 {
+    text-align: center;
+}
+
+input, input:focus{
+    border-bottom: none !important;
+    box-shadow: none !important;
+    margin: 10px 0 0px 0 !important;
+}
+
+a:link, a:visited, a:hover, a:active {
+    color: #01A8E8;
+}
+
+.btn {
+    font-size: 15px;
+    color: #FFFFFF;
+    letter-spacing: 0;
+    background: #1CA9E5;
+    border-radius: 100px;
+}
+
+.input-field {
+    font-family: Proxima Nova;
+    font-size: 16px;
+    color: #A7B2B9;
+    letter-spacing: 1.14px;
+    line-height: 16px;
+    border: 1px solid #B4B4B4;
+    border-radius: 100px;
+    padding: 0 0.75rem;
+}
+.input-field-error {
+    border-color: orange;
+}
+.input-field-error label{
+    color: orange;
+}
+.input-field-error:focus-within {
+    border-color: orange;
+}
+.input-field label {
+    left: 0.75rem;
+}
+
+.input-field:focus-within {
+    border: 1px solid #1ba9e5;
+}
+.input-field-error:focus-within {
+    border-color: orange;
+}
+
+.has-error .help-block{
+    color: orange;
+}
+
+
+[type="checkbox"].filled-in + label:after {
+    border-radius: 25px;
+}
+
+.input-field label:not(.label-icon).active {
+    transform: translateY(8px) scale(0.8);
+    transform-origin: 0 0;
+}
+
+.input-field label {
+    transform: translateY(18px);
+}
+
+.center {
+    margin: auto;
+    width: 35%;
+    min-width: 250px;
+}
+
+/* Mobile/desktop dependent: */
+
+@media screen and (min-width: 480px) {
+    .background{
+        background: url(https://owncloud.hpi.de/index.php/s/siSJzCTBN5U3pbf/download) no-repeat center center fixed;
+        -webkit-background-size: cover;
+        -moz-background-size: cover;
+        -o-background-size: cover;
+        background-size: cover;
+        background-repeat: none;
```

Comment:
The value of background-repeat got set already in line 81.
Additionally none is not a valid value for the background-repeat property. See https://www.w3schools.com/cssref/pr_background-repeat.asp supported property values.

Reply:
Fixed this in #355.